addons/marketing_campaign/tests/test_marketing_campaign.py

```diff
@@ -0,0 +1,109 @@
+# -*- coding: utf-8 -*-
+
+from openerp.tests import common
+
+
+class TestMarketingCampaign(common.TransactionCase):
+
+    def test_00_marketing_campaign_tests(self):
+
+        # In order to test process of compaign, I start compaign.
+        partnerchannel = self.env.ref(
+            'marketing_campaign.marketing_campaign_openerppartnerchannel')
+        partnerchannel.signal_workflow('state_running_set')
+
+        # I check the campaign on Running mode after started.
+        self.assertEqual(partnerchannel.state, 'running')
```

Comment:
what happen if partnerchannel state is not running. ? give assert message like : the campaign should be on Running mode after started.